packages/ui/animations/AnimatedList.tsx

```diff
@@ -0,0 +1,104 @@
+import { Box, chakra } from '@chakra-ui/react';
+import React, { PropsWithChildren } from 'react';
+import { slideIn, slideOut } from './keyframes';
+import { useAnimatedList } from './useAnimatedList';
+import { useSize } from './useSize';
+
+interface AnimatedListProps<T = any> {
+   items: T[];
+   renderItem: (item: T) => React.ReactNode;
+   getItemId?: (item: T) => string;
+   debug?: string;
```

Comment:
Is this a debugging artifact? I don't see it used

Reply:
Yes, I was using that while developing the animations, I thought I dropped it all, I'm quite surprise that this one managed to survived because I've reviewed the code multiple times before submitting the review 😅